integration/global_test.go

```diff
@@ -0,0 +1,87 @@
+package integration_test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testGlobal(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect     = NewWithT(t).Expect
+		Eventually = NewWithT(t).Eventually
+
+		pack   occam.Pack
+		docker occam.Docker
+	)
+
+	it.Before(func() {
+		pack = occam.NewPack().WithVerbose().WithNoColor()
+		docker = occam.NewDocker()
+	})
+
+	context("with global installation", func() {
+		var (
+			image     occam.Image
+			container occam.Container
+
+			name   string
+			source string
+		)
+
+		it.Before(func() {
+			var err error
+			name, err = occam.RandomName()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(docker.Container.Remove.Execute(container.ID)).To(Succeed())
+			Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
+			Expect(docker.Volume.Remove.Execute(occam.CacheVolumeNames(name))).To(Succeed())
+			Expect(os.RemoveAll(source)).To(Succeed())
+		})
+
+		it("builds and runs", func() {
+			var err error
+			var logs fmt.Stringer
+
+			source, err = occam.Source(filepath.Join("testdata", "default_app_global"))
+			Expect(err).NotTo(HaveOccurred())
+
+			image, logs, err = pack.Build.
+				WithPullPolicy("never").
+				WithBuildpacks(buildpacksArray...).
+				WithEnv(map[string]string{
+					"BP_COMPOSER_INSTALL_GLOBAL": "friendsofphp/php-cs-fixer",
+					"BP_LOG_LEVEL":               "DEBUG",
+					"BP_PHP_SERVER":              "nginx",
+				}).
+				Execute(name, source)
+			Expect(err).ToNot(HaveOccurred(), logs.String)
+
+			Expect(logs).To(ContainLines(ContainSubstring("Paketo PHP Distribution Buildpack")))
```

Comment:
Would we also expect to see logs about `Running 'composer global require` and related global install events? Just to be sure that the global behaviour is happening.

Reply:
This sentiment holds true for the other integration tests too

Reply:
done

Reply:
done